problem_builder/public/js/instructor_tool.js

```diff
@@ -250,6 +250,111 @@ function InstructorToolBlock(runtime, element) {
         if (statusChanged) updateView();
     }
 
+    // Block types with answers we can export
+    var questionBlockTypes = ['pb-mcq', 'pb-rating', 'pb-answer'];
+
+    // Fetch this course's blocks from the REST API, and add them to the
+    // list of blocks in the Section/Question drop-down list.
+    function getCourseBlocks() {
+        $.ajax({
+            type: 'GET',
+            url: $rootBlockId.data('course-blocks-api'),
+            data: {
+                course_id: $element.data('course-id'),
+                requested_fields: 'name,display_name,block_type,children',
+                student_view_data: questionBlockTypes.join(','),
+                all_blocks: true,
+                depth: 'all'
+            },
+            success: updateBlockOptions,
+            dataType: 'json'
+        });
+    }
+
+    // Appends the blocks returned by the Course Blocks API as options for
+    // the Section/Question drop-down list, arranged as a tree.
+    function updateBlockOptions(data) {
+
+        // Constructs an <option> element from the given block to add to the
+        // list of root blocks.
+        //
+        // Uses the block's:
+        // * question, name, or display name as the label.
+        // * depth in the course to indent the label, to make the tree
+        //   structure more visible.
+        // * 'enabled' attribute to decide whether the <option>
+        //    element is selectable, i.e. available as a download filter.
+        //
+        // Returns the <option> element so that it can be enabled later,
+        // if it's found to have a descendant that is enabled.
+        var appendBlock = function(block) {
+            var blockId = block.id.split('+block@').pop(),
+                padding = Array(block.depth).join('&nbsp;'),
```

Comment:
@pomegranited Can you increase the padding to two spaces? That's what the original implementation used (see [instructor_tool.html#L35](https://github.com/open-craft/problem-builder/pull/129/files#diff-50b3d2973b892ed1e63a7e85503230fbL35)), and I think the tree looks slightly better/more readable with two spaces.

Sorry for not pointing this out sooner, I really had to see the two versions side-by-side to get a sense of the difference.